patches/server/0066-New-Network-System.patch

```diff
@@ -0,0 +1,307 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Hugo Planque <hookwood@pop-os.localdomain>
+Date: Mon, 18 Jan 2021 11:27:08 +0100
+Subject: [PATCH] New Network System
+
+Co-authored-by: Ivan Pekov <ivan@mrivanplays.com>
+
+diff --git a/pom.xml b/pom.xml
+index 8af1a91102c5cc4c230f622e6629e46e95f17d44..9b7ed0de1054285dadff6aefc95c7207079504a6 100644
+--- a/pom.xml
++++ b/pom.xml
+@@ -53,9 +53,17 @@
+         <dependency>
+             <groupId>io.netty</groupId>
+             <artifactId>netty-all</artifactId>
+-            <version>4.1.50.Final</version>
++            <version>4.1.58.Final</version> <!-- Yatopia -->
+         </dependency>
+         <!-- Tuinity end - fix compile issue (cannot see new api) by moving netty include BEFORE server jar -->
++        <!-- Yatopia start - Add IOUring beta support -->
++        <dependency>
++            <groupId>io.netty.incubator</groupId>
++            <artifactId>netty-incubator-transport-native-io_uring</artifactId>
++            <version>0.0.3.Final</version>
++            <classifier>linux-x86_64</classifier>
++        </dependency>
++        <!-- Yatopia end -->
+         <dependency>
+             <groupId>io.papermc</groupId>
+             <artifactId>minecraft-server</artifactId>
```

Comment:
Shouldn't these pom changes be in `0001-Modify-POM.patch`?

Reply:
no

Reply:
Why not? At least bumping the netty version should be.

Reply:
> Why not?

patch related stuff should be in the patch itself.

> At least bumping the netty version should be.

it is not a necessarity.